### DISCUSSION TITLE
Double login issue

### DISCUSSION THREAD

**Discussion by LinuxServers** [games647/FastLogin#705] [General]
So today I was setting up FastLogin and everything worked fine but when I join the server I get teleported to spawn point but after 1-2 seconds I get teleported again but to AuthMe spawn. I don't know how to fix this, because it feels like something is blocking direct teleport to AuthMe spawn.

**Comment by TuxCoding** [comment]
It's because AuthMe starts another login attempt after FastLogin. You could disable session logins to work around this.
